feat: home-page

BelideNikhil opened the pull request:
Added=>Homepage

Divided everything into components

web view

![image](https://user-images.githubusercontent.com/66310861/159082781-ded75a36-6761-4d49-9648-23494ce7bf2d.png)


mobile view

![image](https://user-images.githubusercontent.com/66310861/159082899-4b82024a-1083-4fe7-a5e0-bc4e1a44bbb4.png)


Preview: https://ninty-seven-x-store.netlify.app/

BelideNikhil committed:
feat: home-page

srishti-maurya commented on src/Components/BestSellers/BestSellerCard.jsx:
```diff
@@ -0,0 +1,19 @@
+    return (
+        <div className="card card-with-badge best-seller-card">
+            <div className="card-image">
+                <img src={source} alt="bestSeller" />
```
you may give a dynamic alt too, so that it is not same always

BelideNikhil replied:
Will do it once the backend data is in place

srishti-maurya commented on src/Components/BestSellers/BestSellers.jsx:
```diff
@@ -0,0 +1,30 @@
+import BestSellerCard from "./BestSellerCard"
+import { v4 as uuid } from "uuid";
+
+const bestSellerData=[{
```
you might add this dataset to backend folder maybe

BelideNikhil replied:
I'll do that Srishti, once I add data to the backend

srishti-maurya commented on src/Pages/Home/Home.jsx:
```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import Navbar from "../../Components/Navbar/Navbar";
+import NavAside from "../../Components/NavAside/NavAside";
+import HeroSection from "../../Components/HeroSection/HeroSection";
+import Categories from "../../Components/Categories/Categories";
+import BestSellers from "../../Components/BestSellers/BestSellers";
```
you might use barrel export

BelideNikhil replied:
done

gopalbbharadva reviewed (approved):
As srishti suggested you can use barrel export. Everything else is fine.

AbhishekkGautam reviewed (approved):
Home ui design is great. Code is also good. We can use barrel exports to make the code more cleaner.

BelideNikhil committed:
refactor: barrel-export

BelideNikhil commented:
> Home ui design is great. Code is also good. We can use barrel exports to make the code more cleaner.

thanks man.
refactored the code and added barrel exports
